app/Services/VocabularyServiceProvider.js

```diff
@@ -93,8 +95,13 @@ async function getGroupVocabulary(userId, groupId) {
     ],
     attributes: ['id', 'name', 'active', 'description'],
     where: {
-      userId,
-      groupId,
+      [Op.and]: [
+        { userId, groupId },
+        search &&
+          sequelize.where(sequelize.fn('lower', sequelize.col('VocabularyCard.name')), {
```

Comment:
I think you should add an or statement to also include `VocabularyCard.Translations.name` in the search.

Reply:
✅